integration-tests/regression.test.js

```diff
@@ -0,0 +1,39 @@
+const { getByTestId, fireEvent, wait } = require('@testing-library/dom')
+const { start } = require('../src/tram-one')
+
+const { updatablePromisePage, conflictingGlobalCounter } = require('./mock-components')
+
+describe('Tram-One - regressions', () => {
```

Comment:
this file represents things that aren't necessarily features, but they are requirements around how tram-one should act... really it comes out of tram-one acting in strange ways that are bugs, and we want to make sure they don't happen again -> ergo regressions